bluepyefe/ecode/sAHP.py

```diff
@@ -32,6 +32,9 @@ class SAHP(Recording):
 
     """sAHP current stimulus
 
+    The long step (here amp) is usually fixed at 40% of rheobase, and the short step (here amp2)
```

Comment:
Do you want to put default value of 40% to the parameter in ecode so when it is not provided it is taken by default? Similarly for other ecodes default values

Reply:
I thought about it, but the problem is that if a user is doing a threshold based opt, but is, for some reason, not using a 40% long step, then the default 40% threshold amp will be used instead of the user-input experimental absolute mean long amp.

Same for hyperdepol and dehyperpol

Reply:
Sorry, I mixed up the PRs. I can put the default values, but they would be overwritten anyway after we compute rheobase. Do you want me to put them nonetheless?

Reply:
Ok, I'll merge this and do a new PR later